server/game/cards/16-DT2/GhostlyCommunion2.js

```diff
@@ -15,7 +15,7 @@ class GhostlyCommunion2 extends SpellCard {
                     location: 'play area',
                     condition: card => card.hasKeyword('holy ground') || 
                         card.adjacentLocations().some(location => location.locationCard.hasKeyword('holy ground')) ||
-                        card.isNearby(this.controller.outfit.uuid)
+                        card.isNearby(this.controller.outfit.uuid) || this.game.townsquare.uuid
```

Comment:
this change would always result in all adjacent locations of every holy ground location to become legal targets. since `this.game.townsquare.uuid` is `townsquare`, which is a truthy value so this condition would always return OK for any card given.

to check if the given card represents the townsquare, you could do a comparison like that instead:

`|| this.game.townsquare.uuid === card.uuid`